d2bs/kolbot/libs/common/Pather.js

```diff
@@ -617,6 +617,94 @@ ModeLoop:
 					/* i < areas.length - 1 is for crossing multiple areas.
 						In that case we must use the exit before the last area.
 					*/
+					if (targetArea == 65) {
+						unit = getUnit(2, 74);
+						if (unit) {
+							switch (unit.mode) {
```

Comment:
switch case format is not consistent,
See lines 709 from your changes or line 621 from original changes.

Reply:
Updated in latest commit, wasn't paying attention.